app/components/provider_interface/filter_component.rb

```diff
@@ -0,0 +1,22 @@
+module ProviderInterface
+  class FilterComponent < ActionView::Component::Base
+    include ViewHelper
+
+    attr_reader :sort_order, :current_sort_by, :filter_options
+
+    def initialize(page_state:)
+      @sort_order = page_state.sort_order
+      @current_sort_by = page_state.sort_by
+      @filter_options = page_state.filter_options
+    end
+
+    def status_filter_checkbox_names_text
+      [
+        ["pending_conditions", "Accepted"], ["recruited", "Conditions met"],
+        ["declined", "Declined"], ["awaiting_provider_decision", "New"],
+        ["offer", "Offered"], ["rejected", "Rejected"],
+        ["withdrawn", "Application withdrawn"], ["offer_withdrawn", "Offer withdrawn"]
```

Comment:
Are the human-readable labels already in a YAML file somewhere? If not would that be a better place for them?